app/assets/stylesheets/spotlight/_featured_browse_categories_block.scss

```diff
@@ -0,0 +1,90 @@
+$featured-browse-category-border-color: #b3b3b3;
+$featured-browse-category-caption-color: #f7f7f7;
+$smallest-featured-browse-category-width: 220px;
```

Comment:
Do you think we can calculate these from $container-tablet / $container-desktop / $container-large-desktop?